.github/workflows/clang-tidy.yml

```
@@ -0,0 +1,33 @@
+name: clang-tidy
+on:
+  pull_request:
+    - main
```

actionlint (1.7.12)

[error] "pull_request" section is sequence node but mapping node is expected

(syntax-check)